src/api/app/controllers/webui/request_controller.rb

```diff
@@ -139,10 +139,13 @@ def show
   def request_action
     @diff_limit = params[:full_diff] ? 0 : nil
     @index = params[:index].to_i
-    @actions = @bs_request.webui_actions(filelimit: @diff_limit, tarlimit: @diff_limit, diff_to_superseded: @diff_to_superseded, diffs: true, action_id: params['id'].to_i)
+    @actions = @bs_request.webui_actions(filelimit: @diff_limit, tarlimit: @diff_limit, diff_to_superseded: @diff_to_superseded, diffs: true,
+                                         action_id: params['id'].to_i, cacheonly: 1)
     @action = @actions.find { |action| action[:id] == params['id'].to_i }
     @active = @action[:name]
     @not_full_diff = BsRequest.truncated_diffs?(@actions)
+    errors = @action[:sourcediff]&.map { |a| a[:error] }&.compact
+    @refresh = errors.present? && errors.any? { |e| e.include?('cache') }
```

Comment:
This sounds a bit brittle to me. If there ever will be any other error message that contains this string...

Reply:
I'm now using the whole error message to verify the correct error.